README.md

```diff
@@ -1,226 +1,51 @@
-# potentially alter/delete existing 3scale elements **
-export NAMESPACE="operator-test"
-oc new-project $NAMESPACE
-oc project $NAMESPACE
-
-# Create the 3scale-operator ServiceAccount
-oc create -f deploy/service_account.yaml
-
-# Create the roles and role bindings associated to the 3scale-operator
-# to be deployed
-oc create -f deploy/role.yaml
-oc create -f deploy/role_binding.yaml
-
-# Set the desired operator image in the operator YAML. For example,
-# the latest available one
-sed -i 's|REPLACE_IMAGE|quay.io/3scale/3scale-operator:latest|g' deploy/operator.yaml
-
-# Deploy the 3scale-operator in the created project
-oc create -f deploy/operator.yaml
-
-# Verify that the operator is deployed and ready. Execute the
-# following command until it shows the Deployment is ready
-oc get deployment 3scale-operator
-```
-
-At this point, the 3scale-operator is deployed and ready to accept the
-3scale-operator custom resource creation requests that it can process to
-perform the provide the functionalities described in the [Overview](#Overview)
-section.
-
-To see a deploy example of the 3scale-operator, how to deploy example custom
-resources to deploy a 3scale API Management solution and 3scale API definitions,
-and how to cleanup the operator and the custom resources
-refer to the [User guide](doc/user-guide.md).
-
-## Development
-
-Assuming you have already downloaded the 3scale-operator project (see
-[Getting Started](#Getting-started)), and your workspace meets [prerequisites](#Prerequisites),
-you can easily build and test the operator:
+The 3scale Operator creates and maintains the Red Hat 3scale API Management on [OpenShift](https://www.openshift.com/) in various deployment configurations.
+[3scale API Management](https://www.redhat.com/en/technologies/jboss-middleware/3scale) makes it easy to manage your APIs.
+Share, secure, distribute, control, and monetize your APIs on an infrastructure platform built for performance, customer control, and future growth.
 
-### Build operator
+## Quickstart
 
-Install dependencies
+To get up and running quickly, check our [Quickstart guides](doc/quickstart-guide.md).
 
-```sh
-# Activate Go Modules
-export GO111MODULE=on
-make vendor
-```
+## Features
 
-Build docker image with the operator installed. Docker image is not pushed to any image repository.
+Current *capabilities* state: **Full Lifecycle**
 
-```sh
-make build IMAGE=quay.io/myorg/3scale-operator VERSION=test
-```
+* Stable
+  * **Installer**: A way to install a 3scale API Management solution, providing configurability options at the time of installation
+  * **Upgrade**: Upgrade from previously installed 3scale API Management solution
+  * **Reconciliation**: Tunable CRD parameters after 3scale API Management solution has been installed
+* Tech Preview
+* **Feature Operator**: Allow interacting with underlying 3scale API Management solution. Expose objects like *tenants*, *api*, *metric*, *limits* as _Custom Resource_ objects.
```

Comment:
>  **Feature Operator**

Maybe we should call it "Capabilities operator"? Last feedback I received was that we should call it that way because it is being identified that way among middleware products. If we don't need to name it that way an alternative might be "API Management interaction"

Reply:
I think this should be indented right?

Reply:
I have seen it as "Feature Operator" in some RedHat presentation about operators, referring to an operator with those advanced capabilities. Capabilities Operator is not a good name IMO. But I will change to be consistent with middleware

:+1:

Reply:
`Application Capabilities via Operator` sounds ok?

Reply:
:+1: